src/Workspaces/CoreTest/SolutionTests/ProjectInfoTests.cs

```diff
@@ -26,17 +26,36 @@ public void Create_Errors()
             Assert.Throws<ArgumentNullException>(() => ProjectInfo.Create(pid, VersionStamp.Default, name: "Goo", assemblyName: "Bar", language: "C#",
                 documents: new DocumentInfo[] { null }));
 
+            var documentInfo = DocumentInfo.Create(DocumentId.CreateNewId(pid), "doc");
```

Comment:
These are all testing duplicates? Can we break out the test a bit or add some comments? This is otherwise turning into the grab bag of random indecipherable asserts...